users/forms.py

```diff
@@ -18,6 +20,28 @@ class Meta:
         fields = ("email",)
 
 
+class SimpleUserChangeForm(ModelForm):
+    """
+    A form for updating users by EMPLOYEE users..
+    """
+
+    class Meta:
+        model = CustomUser
+        fields = ("first_name", "last_name", "date_of_birth", "phone_number", "country")
+        widgets = {"country": CountrySelectWidget(), "date_of_birth": DatePickerInput(options={"format": "YYYY-MM-DD"})}
```

Comment:
Please use constant for "YYYY-MM-DD" format, I think its already defined somewhere (or in settings?).